test/contracts/core/modules/rebalancingSetCTokenExchangeIssuanceModule.spec.ts

```diff
@@ -1454,14 +1461,17 @@ contract('RebalancingSetExchangeIssuanceModule', accounts => {
       });
 
       it('returns the user the leftover receive token amount', async () => {
```

Comment:
Can we have a test that does the baseSet component and then another one for the underlying component?